From 3052556bd558bb2ae48269d115059ca41d77b2f2 Mon Sep 17 00:00:00 2001
From: chewitt <github@chrishewitt.net>
Date: Mon, 29 Apr 2019 03:37:28 +0000
Subject: [PATCH 167/182] WIP: arm64: dts: meson-gxl-s905x-khadas-vim: use
 rc-khadas remote

Khadas shipped the first batch of VIM devices with the 'geekbox' remote from
another of their brands. All further shipments have used a 'khadas' branded
remote that provides an Android mouse function instead of KEY_SCREEN. As the
IR keycode for the geekbox screen button is different to the khadas mouse
button KEY_SCREEN doesn't work on the Khadas remote, so swap to the rc-khadas
keymap to gain an extra button mapped to KEY_MUTE.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxl-s905x-khadas-vim.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxl-s905x-khadas-vim.dts b/arch/arm64/boot/dts/amlogic/meson-gxl-s905x-khadas-vim.dts
index dc51403..2cb377e 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxl-s905x-khadas-vim.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-gxl-s905x-khadas-vim.dts
@@ -147,7 +147,7 @@
 };
 
 &ir {
-	linux,rc-map-name = "rc-geekbox";
+	linux,rc-map-name = "rc-khadas";
 };
 
 &gpio_ao {
-- 
2.7.4

